src/adapters/aol.js

```diff
@@ -69,7 +56,7 @@ var AolAdapter = function AolAdapter() {
    * @param {ADTECHContext} context the context passed from aol
    */
   function _addBid(response, context) {
-    var bid = bidsMap[context.placement];
+    var bid = bidmanager.getPlacementIdByCBIdentifer(context.alias);
```

Comment:
is this method/map underneath it supposed to be `string => string`? I guess name is kind of confusing. Should we have a different map in the `bidmanager`/what is advantage over a map w/i this module?